src/components/Accordion.tsx

```diff
@@ -0,0 +1,44 @@
+import { Css, IconButton, useTestIds } from "src/index";
+import { ReactNode, useState } from "react";
+
+export interface AccordionProps {
+  title: string;
+  children: ReactNode;
+  disabled?: boolean;
+  defaultExpanded?: boolean;
+  bottomSection?: boolean;
+}
+
+export function Accordion(props: AccordionProps) {
+  const { title, children, disabled = false, defaultExpanded = false, bottomSection = false} = props;
+  const [expanded, setExpanded] = useState(defaultExpanded);
+  const [focused, setFocused] = useState(false);
+  const testIds = useTestIds({}, "accordion");
+
+  return (
+    <div
+      {...testIds}
+      onBlur={() => setFocused(false)}
+      onFocus={() => setFocused(true)}
+      // Remove default border when focused
+      css={Css.if(!focused).bt.bGray400.if(bottomSection && !(!expanded && focused)).bb.$}
+    >
+      <div {...testIds.summary} css={Css.df.if(focused).ba.bLightBlue600.bw2.addIn(":hover", Css.bgGray100.$).$}>
```

Comment:
**suggestion:** Use a `box-shadow` to provide the focus outline instead of a border. This will stop the content from shifting when you introduce the focus styles.

Reply:
(probably something like: `box-shadow: inset 0 0 0 2px Palette.LightBlue600`.